src/protosym/simplecas.py

```diff
@@ -54,7 +54,34 @@ class ExpressifyError(TypeError):
 
 
 def expressify(obj: Any) -> Expr:
-    """Convert a native object to an ``Expr``."""
+    """Convert a native Python object to an ``Expr``.
+
+    >>> from protosym.simplecas import expressify
+    >>> one = expressify(1)
+    >>> one
+    1
+    >>> type(one)
+    <class 'protosym.simplecas.Expr'>
+
+    This is the internal representation of the ``one`` object returned by
+    :func:`expressify`:
+
+    >>> one.rep
+    TreeAtom(Integer(1))
+
+    It is harmless ro call :func:`expressify` more than once because it will
```

Comment:
```suggestion
    It is harmless to call :func:`expressify` more than once because it will
```

Reply:
Feel free to just commit changes like this.